### PR TITLE
feat(unlock-app) - refresh members page ui

### PR DESCRIPTION
**Pull request by kalidiagne**
# Description
- This PR add new UI to members page, in particular we have now cards instead of tables to better show extra data. Please see the following screens. 

- [x] Add button to show all metadata for all members at once
- [x] new UI with old functionalities 

# Issues

Fixes #
Refs #

#### component UI refreshed 
Screenshot 2022-06-22 at 20 49 18

#### basic list
Screenshot 2022-06-22 at 22 51 15

#### basic list with metadata opened for a specific key 
Screenshot 2022-06-22 at 22 51 24

#### Show all metadata button
Screenshot 2022-06-23 at 11 55 41

### Responsive
Screenshot 2022-06-24 at 11 16 52

# Checklist:

- [x] 1 PR, 1 purpose: my Pull Request applies to a single purpose
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have updated the docs to reflect my changes if applicable
- [ ] I have added tests (and stories for frontend components) that prove my fix is effective or that my feature works
- [x] I have performed a self-review of my own code
- [x] If my code involves visual changes, I am adding applicable screenshots to this thread

### CODE REVIEW

File: unlock-app/src/components/interface/members/MemberCard.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+  lockName,
+  expiration,
+  keyholderAddress,
+  tokenId,
+  onExpireAndRefund,
+  expireAndRefundDisabled = true,
+  metadata = {},
+}) => {
+  const [showMetaData, setShowMetaData] = useState(false)
+  const addressMinify = (address: string): string => {
```
Comment (searchableguy) [inline unlock-app/src/components/interface/members/MemberCard.tsx:32]: move this into utilities? Seem like something we need everywhere.

File: unlock-app/src/components/interface/members/MemberCard.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+        </div>
+        <div className="flex flex-col">
+          <span className={styles.description}>Expiration</span>
+          <span className={styles.title}>{expiration}</span>
+        </div>
+        <div className="flex flex-col">
+          <span className={styles.description}>Checked in At</span>
+          <span className={styles.title}>-</span>
+        </div>
+        <div className="flex gap-1">
```
Comment (searchableguy) [inline unlock-app/src/components/interface/members/MemberCard.tsx:75]:
```suggestion
        <div className="flex gap-2">
```

File: unlock-app/src/components/interface/members/MemberCard.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react'
+import { Button } from '@unlock-protocol/ui'
+
+const styles = {
+  title: 'text-lg font-bold text-black',
+  description: 'text-xs	font-sm font-normal',
```
Comment (searchableguy) [inline unlock-app/src/components/interface/members/MemberCard.tsx:6]:
```suggestion
  description: 'text-xs font-normal text-gray-500',
```

File: unlock-app/src/components/interface/members/MemberCard.tsx [hunk 4]
```diff
@@ -0,0 +1,114 @@
+import React, { useState } from 'react'
+import { Button } from '@unlock-protocol/ui'
+
+const styles = {
+  title: 'text-lg font-bold text-black',
```
Comment (searchableguy) [inline unlock-app/src/components/interface/members/MemberCard.tsx:5]:
```suggestion
  title: 'text-base font-medium text-black',
```

**Comment by searchableguy** [comment]
ideally, this should be squareish with two column layout for items inside or you should set possible space values for each column (you can do this using grid and col-span) so everything looks aligned. 

I would prefer the former option but I think it's "good" enough for now.

**Comment by kalidiagne** [comment]
@searchableguy updated with grid for better alignment, for the rest nice points to add !

**Comment by searchableguy** [comment]
Adjust columns for different screen size please.

**Comment by julien51** [comment]
O hwow, that's looking really good!

**Comment by julien51** [comment]
I really like where this is going! 2 things:
1. Can you add a button at the top to show _all_ metadata for all members at once (expands all) but only show this button if the "viewer" is a lock manager
2. Do not show the `Checked In at` column if none of token has a checked in time.

**Comment by kalidiagne** [comment]
- Added responsive 
- "show all metadata"  to expand all members at once 
I will open a new PR to add check-in status value and show it  only if none of token has a checked in time

**Pull request merged**